### PR TITLE
Adds welcome page

### PR DESCRIPTION
**Pull request by Olamuhammed**
This just adds a welcome page route

### CODE REVIEW

**Review by kdoran — CHANGES_REQUESTED** [review]
Nice work m-man 🎉 first PR :)

few things to try out:

1. your welcome file only needs one import :)
2. try running the command `standard` and see what errors it says, it's an npm script 
3. there's a convention for file names that are "containers" (which we use as controllers of sorts) which is `FileNameContainer`, also all the imports for containers in the same place in the app file

File: app/app.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import withParams from './containers/withParams'
 import cache from './utils/cache'
 import { parseUrl } from './utils/utils'
 import fetcher from 'utils/fetcher'
+import Welcome from './containers/WelcomeContainer'
```
Comment (kdoran) [inline app/app.js]: you'd still want this to be called `WelcomeContainer` to match the others (doesn't really matter but for the sake of convention).

File: app/app.js [hunk 2]
```diff
@@ -88,6 +90,7 @@ class App extends Component {
         <div>
           <Route exact path='/' component={SetupCouchContainer} />
           <Route path='/:couch/' component={UserRoutes} />
+          <Route path='/Welcome' component={Welcome} />
```
Comment (kdoran) [inline app/app.js]: the route would be lowercase `welcome`

File: app/components/Welcome.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, {Component} from  'react';
+import ReactDOM from 'react-dom';
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+
+
+class Welcome extends Component{
```
Comment (kdoran) [inline app/components/Welcome.js]: same here keep the naming uniform `WelcomeContainer`

File: app/containers/SetupCouchContainer.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export default class extends React.Component {
 
   async tryACouch (inputUrl) {
     inputUrl = parseUrl(inputUrl)
-    this.setState({inputUrl})
+    this.setState({ inputUrl })
```
Comment (kdoran) [inline app/containers/SetupCouchContainer.js]: lol you don't like my newfound lack of spaces in object destructuring ;)

File: app/containers/SetupCouchContainer.js [hunk 5]
```diff
@@ -81,7 +81,7 @@ export default class extends React.Component {
         >
           {loading ? ' Loading...' : 'Submit'}
         </button>
-        <a href='https://github.com/kdoran/lookout'>Lookout on GitHub</a>
+        <a href='https://github.com/kdoran/lookout'>Lookout on GitHub</a><br />
```
Comment (kdoran) [inline app/containers/SetupCouchContainer.js]: what's this little `<br />` guy about

File: app/containers/WelcomeContainer.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
```
Comment (kdoran) [inline app/containers/WelcomeContainer.js]: there are two of these files now :)

**Comment by Olamuhammed** [comment]
Hey, made the changes to name and deleted somethings too

**Pull request closed without merging**